jawsManagerApp/Db/jaws_manager_single.template

```diff
@@ -0,0 +1,72 @@
+# Jaws manager single jawset
+#
+#  Records for a single jawset in the jawsmanager. Provides the maths for the collimation and aliases to create a managed jawset
+#  These aliases provide an abstraction layer for the top manager (see wiki for details)
+#
+#  Macros:
+#    P - base name for ioc
+#    JAW - The jaw that this corresponds to
+#	 UNDERLYING_JAW - The pv for the underlying jaw (defaults to the jaw given by JAW e.g. MOT:JAW1:)
+#    SAMPLE_JAW - the pv for the (potentially imaginary) jawset at the sample
+#    MOD_JAW - the pv for the (potentially imaginary) jawset at the moderator
+#    MOD_TO_SAMPLE - Distance from moderator to sample
+#    MOD_OFFSET - Distance from moderator to an offest position where collimation should start from (Defaults to zero)
+#    MOD_TO_JAW - Distance from moderator to this jaw set
+
+record(transform, "$(P)JAWMAN:$(JAW):HGAP:_CALC") {
+
+  field(INPA, "$(P)$(SAMPLE_JAW=JAWMAN:SAMPLE):HGAP:SP")
+  field(INPB, "$(P)$(MOD_JAW=JAWMAN:MOD):HGAP:SP")
+  field(INPC, "$(MOD_TO_SAMPLE)")
+  field(INPD, "$(MOD_OFFSET=0)")  
+  field(INPE, "$(MOD_TO_JAW)") 
+  
+  field(CLCK, "(A+((B-A)*((C-E)/(C-D))))")
+  
+  field(OUTK, "$(P)JAWMAN:$(JAW):HGAP:SP PP")
+}
+
+record(transform, "$(P)JAWMAN:$(JAW):VGAP:_CALC") {
+
+  field(INPA, "$(P)$(SAMPLE_JAW=JAWMAN:SAMPLE):VGAP:SP")
+  field(INPB, "$(P)$(MOD_JAW=JAWMAN:MOD):VGAP:SP")
+  field(INPC, "$(MOD_TO_SAMPLE)")
+  field(INPD, "$(MOD_OFFSET=0)")  
+  field(INPE, "$(MOD_TO_JAW)") 
+  
+  field(CLCK, "(A+((B-A)*((C-E)/(C-D))))")
+  
+  field(OUTK, "$(P)JAWMAN:$(JAW):VGAP:SP PP")
+}
+
+record(ao, "$(P)JAWMAN:$(JAW):HGAP:SP") {
```

Comment:
Do these records need to be archived? The delayed jaw values are archived

Reply:
They will always match the values in the underlying jaw, which is archived, the delayed version will not match the underlying one until the `SET` PV is called so it makes sense to archive both in that case.